eslint/babel-eslint-parser/package.json

```diff
@@ -1,32 +1,29 @@
 {
-  "name": "babel-eslint",
-  "version": "11.0.0-beta.1",
-  "description": "Custom parser for ESLint",
+  "name": "@babel/eslint-parser",
+  "version": "0.0.0",
+  "description": "ESLint parser that allows for linting of experimental syntax transformed by Babel",
   "author": "Sebastian McKenzie <sebmck@gmail.com>",
   "license": "MIT",
   "private": true,
   "repository": {
     "type": "git",
-    "url": "https://github.com/babel/babel-eslint.git"
+    "url": "https://github.com/babel/babel.git"
```

Comment:
nit: NPM [recommends](https://docs.npmjs.com/files/package.json#repository) to add `directory: "eslint/babel-eslint-parser"` since babel is a monorepo.

Reply:
Cool, didn't know this. Thanks!